packages/screens/ApplicationRewiew/ApplicationRewiew.tsx

```diff
@@ -0,0 +1,648 @@
+import React from "react";
```

Comment:
We should split this file into several components

Reply:
Fixed, PR updated